core/local/steps/incomplete_fixer.js

```diff
@@ -27,6 +31,22 @@ module.exports = {
   loop
 }
 
+function itemDestinationWasRenamed (item /*: IncompleteItem */, event /*: AtomWatcherEvent */) /*: boolean %checks */ {
+  return !!(
+    event.action === 'renamed' &&
+    event.oldPath &&
+    item.event.path.startsWith(event.oldPath + path.sep)
+  )
+}
+
+function itemDestinationWasDeleted (item /*: IncompleteItem */, event /*: AtomWatcherEvent */) /*: boolean %checks */ {
```

Comment:
- Start function name with verb (makes it easier to understand it is a boolean function)
- Rename `item` and `event` to e.g. `previousIncomplete` and `nextEvent` (makes it easier to understand which one precedes the other)
- Rename functions to e.g. `wasRenamedSuccessively() ` and `wasRenamedThenDeleted()`?

Reply:
Partially done in https://github.com/cozy-labs/cozy-desktop/pull/1474